packages/core/template_project/app/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,7 @@
      the abbreviated format. -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="<%= packageId %>">
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
Did this get added in by Android Studio or do we need it for WebView? It's seems a little ugly including it even if we don't need WebView, but on the other hand it's not that much overhead.

Reply:
This needs to be added for the WebView to work on Android 9 and below. We conditionally add it, only when the fallback is type `webview`

Reply:
Missed the "could" on the text. Fixed that.

Reply:
If it's easy enough let's do it, I don't think it's blocking though.

Reply:
Done